f8a_worker/models.py

```diff
@@ -411,3 +411,35 @@ class OSIORegisteredRepos(Base):
     git_sha = Column(String(255), nullable=False)
     email_ids = Column(String(255), nullable=False)
     last_scanned_at = Column(DateTime)
+
+
+class ProbableCVEs(Base):
+    """Table for storing daily Probable CVEs"""
+
+    __tablename__ = "probable_cves"
+
+    id = Column(Integer, autoincrement=True, primary_key=True)
+    ecosystem = Column(String(50), nullable=False)
```

Comment:
Do we need this if this is just for Golang?

Reply:
We are not going to do it for other ecosystems anytime soon.  For now, it will always be "Golang".  However, we have the plan to expand it in future for other ecosystems based on how the model behaves.

Reply:
Is it a good idea to have it there as a string, without any constraints? Wouldn't be better to have for example enum for ecosystem names? I don't have enough context here to say what's the best approach here.

Reply:
+1